test/manual/api/structure_signature_api_tests.sh

```
@@ -0,0 +1,462 @@
+  local list="$1"
+  local item="$2"
+  if [ -z "$list" ]; then
+    echo "$item"
+  else
+    echo "$list $item"
+  fi
+}
+
+# Fetch the first available system (ID and name) from the API
+get_first_system() {
+  local raw=$(make_request GET "$API_BASE_URL/api/maps/$MAP_SLUG/systems")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)